Update gidel18a and supp

GauthierGidel opened the pull request.

Commit:
Update gidel18a and supp

lawrennd commented:
Hi Gauthier,

Could you please explain the updates and provide confirmation from the Program Chairs that you have permission to change the PDFs?

Thanks!

Neil

GauthierGidel commented:
Hi,

I forward you the conversation with Chloe-Agathe Azencott who gave me the
permission to submit a pull request.
As I said to Chloe, the updates are the followings:

*"We got contacted by peers about some  work related to our paper and we
have implemented a related work section in consequence. We also corrected
some typos."*
Best,
Gauthier Gidel


---------- Forwarded message ---------
From: Chloe-Agathe Azencott <chloe-agathe.azencott@mines-paristech.fr>
Date: lun. 9 avr. 2018 à 13:07
Subject: Re: Update of the camera ready version
To: Gauthier Gidel <gauthier.gidel@gmail.com>
Cc: Simon Lacoste-Julien <slacoste@iro.umontreal.ca>, Fabian Pedregosa <
f@bianp.net>


Dear Gauthier,

In order to submit an updated version to your paper, you can click the
"edit" link on the upper right corner of your paper page:
http://proceedings.mlr.press/v84/gidel18a.html

The link will take you to github, where you'll be able to fork the repo.
You will then be able to make edits and submit a pull request.

Let me know if you need help with the process.

Best wishes,
Chloe.

On 04/09/2018 06:32 PM, Gauthier Gidel wrote:
> Hello,
>
> My co-autor and I have a paper accepted AISTATS2018 called "Frank-Wolfe
> Splitting via Augmented Lagrangian Method" (Paper ID: 483).
> May I ask you if it is possible to update our camera-ready version ?
> We got contacted by peers about some  work related to our paper and we
> have implemented a related work section in consequence.
> We also corrected some typos.
> I case you agree to update our camera ready version, please find
> attached the updated version of the main paper and the supplementary
> materials.
>
> Best,
>
> Gauthier Gidel
>
> --
> Gauthier Gidel,
> PhD candidate,
> Montréal Institute for Learning Algorithm.

-- 
Chloe-Agathe Azencott
CBIO Mines ParisTech - Institut Curie - INSERM U900
http://cazencott.info


Le ven. 20 avr. 2018 à 13:46, Neil Lawrence <notifications@github.com> a
écrit :

> Hi Gauthier,
>
> Could you please explain the updates and provide confirmation from the
> Program Chairs that you have permission to change the PDFs?
>
> Thanks!
>
> Neil
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mlresearch/v84/pull/4#issuecomment-383171899>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AHEAGWujC-A-lDC2TAgyvt6eRqoMKYShks5tqh7-gaJpZM4Tdz3X>
> .
>
-- 
Gauthier Gidel,
PhD candidate,
Montréal Institute for Learning Algorithm.

lawrennd commented:
Thanks Gauthier!
